platforms/allwinner-d1/src/dmac/mod.rs

```diff
@@ -177,3 +318,20 @@ pub enum ChannelMode {
     Wait,
     Handshake,
 }
+
```

Comment:
uncomment or remove dead code

Reply:
unfortunately, `forced_target` means we can't have tests in this crate and run them on the host target. i left this here because i wanted to figure out a solution for that later. i can comment about why it's commented out?

Reply:
Sure! that context helps.